vip-helpers/vip-debug-bar-panels.php

```diff
@@ -437,3 +437,77 @@ function log_http_request_result( $response, $context, $transport, $args, $url )
 		}
 	}
 }
+
+class WPCOM_VIP_Debug_Bar_Elasticsearch extends Debug_Bar_Panel {
```

Comment:
Good call - I moved it to the `debug-bar/panels` folder and removed some of the VIP specific stuff, so maybe it can be used as a more standalone plugin in the future.